Source/RapyutaSimulationPlugins/Private/Core/RRProceduralMeshComponent.cpp

```diff
@@ -68,27 +68,23 @@ bool URRProceduralMeshComponent::InitializeMesh(const FString& InMeshFileName)
             else
             {
                 URRThreadUtils::DoAsyncTaskInThread<void>(
-                    [this, InMeshFileName]()
-                    {
+                    [this, InMeshFileName]() {
```

Comment:
This seems to make inline `{` only in lambda, but not for if/else, so abit inconsistent but more essentially, just curious why the change?

Reply:
hmm, not sure it was applied by clang-format of vscode.